sdk/lib/crt/crt.cmake

```diff
@@ -483,6 +483,10 @@ elseif(ARCH STREQUAL "amd64")
     if(MSVC)
         list(APPEND CRT_ASM_SOURCE
             except/amd64/cpp.s)
+        if (CMAKE_C_COMPILER_ID STREQUAL "Clang")
+            list(APPEND CRT_ASM_SOURCE
+                stdlib/clang-alias.s)
```

Comment:
Do we have this file? (I don't see it?)

Reply:
Yes we do: https://github.com/reactos/reactos/blob/master/sdk/lib/crt/stdlib/clang-alias.s